runelite-client/src/main/java/net/runelite/client/plugins/minimap/MinimapPlugin.java

```diff
@@ -50,6 +55,25 @@
 {
 	private static final int NUM_MAPDOTS = 6;
 
+	private static List<WidgetInfo> orbs = ImmutableList
+		.<WidgetInfo>builder()
+		.add(WidgetInfo.MINIMAP_HEALTH_ORB)
+		.add(WidgetInfo.MINIMAP_PRAYER_ORB)
+		.add(WidgetInfo.MINIMAP_RUN_ORB)
+		.add(WidgetInfo.MINIMAP_SPEC_ORB)
+		.build();
+
+	private static int[][] orbDefaultPositions = {
+		{-1, -1, -1, -1},
+		{-1, -1, -1, -1},
+		{-1, -1, -1, -1},
+		{-1, -1, -1, -1}
+	};
+
+	private static int orbHeight = 32;
+	private static int orbXOffset = 1;
```

Comment:
This should be `private static final`, and var name should be uppercased to match.